token/services/interop/htlc/db/sql/tokendbext.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package sql
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+	"runtime/debug"
+
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/db/driver"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/interop/htlc"
+	"github.com/hyperledger-labs/fabric-token-sdk/token/services/logging"
+	"github.com/pkg/errors"
+)
+
+var logger = logging.MustGetLogger("token-sdk.sql.htlc")
+
+type htlcTokenDBExtensionTables struct {
+	Tokens string
+}
+
+type HTLCTokenDBExtension struct {
+	table htlcTokenDBExtensionTables
```

Comment:
There is a dependency on the 'tables' struct here. We can make our lives much easier by removing the feature of the 'table prefixes' everywhere (which I think we should do anyway!). Then, every db just decides by itself in the code what its tables are called.